### PR TITLE
Added information and a corrected typo

### PR DESCRIPTION
**Pull request by Shingapi**
Added a description of how to use a shortcut to access the UNIX shell.
corrected a typo under Arguments that ls can take.
Added information on how the man command works.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -138,11 +138,11 @@ So, we can see that we have several files, directories and a program. Great!
 
 Most programs take additional arguments that control their exact
 behavior. For example, `-F` and `-l` are arguments to `ls`.  The `ls`
-program, like many programs, take a lot of arguments. But how do we
+program, like many programs, takes a lot of arguments. But how do we
 know what the options are to particular commands?
 
 Most commonly used shell programs have a manual. You can access the
-manual using the `man` program. Try entering:
+manual using the `man` program. The man program will display the manual pages of the ls command page by page; use the spacebar to get tot the next page. Try entering:
```
Comment (dpshelio) [inline index.md]: to keep the formatting as  in the rest of the document is preferred to break your line into three lines.

File: index.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ have to download a separate program.
 Mac
 ---  
 On Mac the shell is available through Terminal  
-Applications -> Utilities -> Terminal  
+Applications -> Utilities -> Terminal (A shortcut would be to press CMD+ spacebar for spotlight search then type terminal and press enter.)
```
Comment (dpshelio) [inline index.md]: This line is also longer than the rest of the document.

**Comment by ErinBecker** [comment]
@Shingapi - Thank you so much for this contribution and I'm sorry about formatting issues preventing it from being merged in a good timeframe. We're working now on improving our Maintenance workflow so that issues and PRs don't linger and become outdated.

If you'd like to re-make a PR to address any of these typos that still exist, we'll make sure to get it merged quickly. If you'd prefer, we can hand this off to one of the Maintainers who will be doing a comprehensive read-through soon. Please let us know which you would prefer and if we should close this PR.

**Comment by Shingapi** [comment]
Please hand this off to one of the Maintainers.

On 4 Oct 2017, at 00:24, Erin Becker <notifications@github.com> wrote:

@Shingapi <https://github.com/shingapi> - Thank you so much for this
contribution and I'm sorry about formatting issues preventing it from being
merged in a good timeframe. We're working now on improving our Maintenance
workflow so that issues and PRs don't linger and become outdated.

If you'd like to re-make a PR to address any of these typos that still
exist, we'll make sure to get it merged quickly. If you'd prefer, we can
hand this off to one of the Maintainers who will be doing a comprehensive
read-through soon. Please let us know which you would prefer and if we
should close this PR.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/datacarpentry/shell-genomics/pull/14#issuecomment-333996777>,
or mute the thread

**Comment by ErinBecker** [comment]
Thanks @Shingapi - will do! I'll assign this to myself now and take care of it before our upcoming lesson release. Thanks again for your contributions!

**Comment by ErinBecker** [comment]
Replaced with #101.

**Pull request closed without merging**